tests/remote_resources/xtcp_ping_pong.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
```

Comment:
It's safest to use `#!/usr/bin/env python` as the shebang to avoid issues on machines with python installed elsewhere.